developer-docs-site/docs/standards/fungible-asset.md

```diff
@@ -1,13 +1,13 @@
 ---
-title: "Fungible Asset"
+title: "Aptos Fungible Asset Standard"
 id: "fungible-asset"
 ---
 import ThemedImage from '@theme/ThemedImage';
 import useBaseUrl from '@docusaurus/useBaseUrl';
 
-# Fungible Asset
+# Aptos Fungible Asset Standard
 
-Fungible Assets (FAs) is a core framework component within Aptos that enables the tokenization of various assets, including commodities, real estate, and financial instruments, and facilitate the creation of decentralized financial applications.
+Fungible Asset (FA) is a core framework component within Aptos that enables the tokenization of various assets, including commodities, real estate, and financial instruments, and facilitate the creation of decentralized financial applications.
```

Comment:
```suggestion
A Fungible Asset (FA) is a core framework component within Aptos that enables the tokenization of various assets, including commodities, real estate, and financial instruments, and facilitate the creation of decentralized financial applications.
```

Reads a little odd otherwise

Reply:
@xbtmatt it was meant to be a shortening of "The Aptos Fungible Asset Standard is a core..."
Adding "a" makes it read like we're talking about an individual instance of FA.

Reply:
Yea I wasn't sure how to write it exactly, I think the long form you mentioned is clearest but I'll defer to you on that :p